publications/abstracts/reservoir-computing-abs/background.tex

```diff
@@ -0,0 +1,45 @@
+\section{Background}
+Variability has been the primary drawback for renewable energy sources like
+wind turbines, solar PV, and solar concentrators since their inception. This
+flaw has become more pronounced as renewable penetration on the electricity
+grid increased in recent years. Forecasting electricity production from
+renewable sources is therefore important for successful management of power
+systems \cite{kobylinski_high-resolution_2020}. Recent studies have applied
+\glspl{ann}, specifically multi-layer perceptrons, to the task of net load
+forecasting \cite{kobylinski_high-resolution_2020,dutta_load_2017,lee_development_2016}. These studies made short
+term forecasts of 4-6 hours. Forecasts at this time scale enable load following
+for natural gas plants, but challenges nuclear power plants. Nuclear plants
```

Comment:
Weird grammar/sentence flow in the second half of "Forecasts at this time scale enable load following for natural gas plants, but challenges nuclear power plants". Maybe change to "...but pose challenges for nuclear power plants"? I'm unsure, it just felt weird in the first read.

Reply:
Thanks Gavin, This will be added to issue #70.